README.md

```diff
@@ -1,2 +1,68 @@
-# Ovio.org
-Updated website for [Ovio.org](ovio.org)
+# Ovio website
+
+https://oviohub.com/
+
+A GitHub repo for Ovio static website.
+
+## Development
+
+### Getting started
+
+This project use **[yarn](https://yarnpkg.com/en/)** package manager with **[gatsbyJs](https://www.gatsbyjs.org/)**, a project builder for **[React](https://reactjs.org/)**.
+
+```bash
+yarn
+yarn start
+```
+The website will be available at <http://localhost:8000/>.
+
+### Deployement
+
+To generate the ```public``` folder, run
+
+```bash
+yarn build
+```
+
+### Project structure
+
+```
+website/
+│   README.md
+│   gatsby-config.js               <-- metadata for project title, description, plugins, etc.
+│   gatsby-node.js                 <-- generation of pages in src/Routes.js
+│   package.json                   <-- configuration of the project (dependencies, scripts, ...)
+│   yarn.lock                      <-- auto-generated by yarn to manage dependencies
+│
+└───public/                        <-- auto-generated on "yarn start" command run
+│   │   ....
+|
+└───node_modules/                  <-- auto-generated by yarn to manage dependencies
+│   │   ....
+│
+└───src/
+│   │   Routes.js                  <-- contains informations (url, seo params, ...) of website's pages
+│   │
+|   └───assets/
+│   │   │   ....
+│   │
+|   └───pages/                     <-- needed by gatsby. All files in it will be convert to a page
+│   │   │   ....
+│   │
+|   └───services/                  <-- contains common files or modules for the website
+│   │   │   ....
+│   │
+|   └───components/
+│   │   |
+│   |   |   styledComponents.js      <-- contains the default theme, global constants (colors, ...)
+│   |   |
+│   |   └───layouts/               <-- contains the wrapper of all pages, and fonts. Allow to integrate the seo params
```

Comment:
I would say "contains the wrapper for all pages" (rather than "of").

"Allows integration of the SEO params"